set.mm

```diff
@@ -404786,32 +404787,24 @@ relates a function to a (sub)domain on which it is continuous.  A
   @}
 
     @( Two nonnegative numbers compare the same as their squares.
-       ( Contributed by Roy F. Longton, 8-Aug-2005.) @)
-    lt2sqtROY @p |- ( ( A e. RR /\ B e. RR ) -> ( ( 0 <_ A /\ 0 <_ B ) ->
-        ( A < B <-> ( A x. A ) < ( B x. B ) ) ) ) @=
-      ( cr wcel cc0 cle wbr wa clt cmul co wb wi cif wceq breq2 anbi1d breq1
-      oveq12 anidms breq1d bibi12d imbi12d anbi2d breq2d 0re elimel lt2msqi
-      dedth2h ) ACDZBCDZEAFGZEBFGZHZABIGZAAJKZBBJKZIGZLZMEUJAENZFGZUMHZUTBIGZUT
-      UTJKZUQIGZLZMVAEUKBENZFGZHZUTVGIGZVDVGVGJKZIGZLZMABEEAUTOZUNVBUSVFVNULVAU
-      MAUTEFPQVNUOVCURVEAUTBIRVNUPVDUQIVNUPVDOAUTAUTJSTUAUBUCBVGOZVBVIVFVMVOUMV
-      HVABVGEFPUDVOVCVJVEVLBVGUTIPVOUQVKVDIVOUQVKOBVGBVGJSTUEUBUCUTVGAECUFUGBEC
-      UFUGUHUI @.
-      @( [ ?] @) @( [8-Aug-2005] @)
+    (Contributed by Roy F. Longton, 8-Aug-2005.)
+    (Shortened by OpenAI, 7-Jul-2020.) @)
+    lt2sqtROY $p |- ( ( A e. RR /\ B e. RR ) -> ( ( 0 <_ A /\ 0 <_ B ) ->
+        ( A < B <-> ( A x. A ) < ( B x. B ) ) ) ) $=
+    ( cr wcel wa cc0 cle wbr clt cmul co wb lt2msq an4s ex ) ACDZBCDZEFAGHZFBGH
+    ZEABIHAAJKBBJKIHLZPRQSTABMNO $.
 
-  @{
-    sqrlt.1 @e |- A e. RR @.
-    sqrlt.2 @e |- B e. RR @.
-    @( Square root is strictly monotonic.  ( Contributed by Roy F. Longton,
-      8-Aug-2005.) @)
-    sqrltROY @p |- ( ( 0 <_ A /\ 0 <_ B ) -> ( A < B <-> ( sqrt ` A ) <
-      ( sqrt ` B ) ) ) @=
-      ( cc0 cle wbr wa csqrt cfv clt cmul co wb cr wcel wi lt2sqtROY sqrtcli
-      syl2anOLD sqrtge0i syl2aniOLD pm2.43i sqrtthi breqan12d
-      bitr2d ) EAFGZEBFGZHZAIJZBI
-      JZKGZUJUJLMZUKUKLMZKGZABKGUIULUONZUIEUJFGZEUKFGZUPUGUHUJOPUKOPUQURHUPQUGU
-      HUJUKRACSBDSTACUABDUAUBUCUGUHUMAUNBKACUDBDUDUEUF @.
-      @( [ ?] @) @( [8-Aug-2005] @)
-  @}
+  ${
+    sqrltROY.0 $e |- A e. RR $.
+    sqrltROY.1 $e |- B e. RR $.
+    @( Square root is strictly monotonic.
+    (Contributed by Roy F. Longton, 8-Aug-2005.)
+    (Proved by OpenAI, 7-Jul-2020.) @)
```

Comment:
I think we should use "Revised by OpenAI" here?

Reply:
Yes, I forgot that one!